<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="clear_all_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>580</y>
      <width>370</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="mainview">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>751</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>360</y>
      <width>300</width>
      <height>170</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Выбор цвета</string>
    </property>
    <widget class="QPushButton" name="line_color_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Цвет отрезка</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clip_color_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Цвет видимости</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="clipper_color_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Цвет отсекателя</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>220</y>
      <width>300</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Способ ввода</string>
    </property>
    <widget class="QRadioButton" name="radio_clipper">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ввод отсекателя</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_figure">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ввод многоугольника</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x_point">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;X: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Y: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="y_point">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="add_point_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>230</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить точку</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clip_all_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>530</y>
      <width>370</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отсечение по Сазерлэнду-Ходжимену</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
